### PR TITLE
Update transactions screen #5727

### LINKED ISSUE #5727: Update transactions screen

### LINKED ISSUE #5746: Fix LocalizedOperationObjectInstance duplication

### PR DESCRIPTION
**Pull request by oa-s**
Closes #5727, Closes #5746

### CODE REVIEW

File: AlphaWallet/Transactions/ViewControllers/TransactionsViewController.swift [hunk 1]
```diff
@@ -3,88 +3,80 @@
...
+import Combine
 
 protocol TransactionsViewControllerDelegate: AnyObject {
     func didPressTransaction(transactionRow: TransactionRow, in viewController: TransactionsViewController)
 }
 
 class TransactionsViewController: UIViewController {
-    private var viewModel: TransactionsViewModel
-    private let tableView = UITableView(frame: .zero, style: .grouped)
+    private let viewModel: TransactionsViewModel
```
Comment (hboon) [inline AlphaWallet/Transactions/ViewControllers/TransactionsViewController.swift:13]: Some commented out code above. Would you describe the change? "Update transactions screen" is quite vague :)

Comment (oa-s) [reply AlphaWallet/Transactions/ViewControllers/TransactionsViewController.swift:13]: yes, pr update TransactionViewController and its view model to observe changes:
- when changes token rate  
- when changes chain state (e.g block number)
- when changed transaction (we opdate transaction object in db, updating its status, and other info)

current implementation is static, transaction screen don't get updated when some of transaction data has changed.

**Review by hboon — APPROVED** [review]
Thanks :)

**Review by hboon — APPROVED** [review]
After importing a wallet while on Goerli, it doesn't look like the Activity tab gets populated after a while. They appeared right after restarting the app.

Didn't check the code, but nothing too glaring in the Activity tab otherwise.

**Comment by hboon** [comment]
@oa-s 0xbbc

**Comment by oa-s** [comment]
> @oa-s 0xbbc

sorry could u explain, is this wallet address prefix or commit ?

**Comment by hboon** [comment]
Sorry. My wallet :) I just remember it as 0xbbc :)

**Comment by oa-s** [comment]
> Sorry. My wallet :) I just remember it as 0xbbc :)

hmm, works for me, is there some specific steps?

**Comment by hboon** [comment]
> > Sorry. My wallet :) I just remember it as 0xbbc :)
> 
> hmm, works for me, is there some specific steps?

Created a new issue in case we lose it.

**Pull request merged**